tests/sentry/tasks/test_beacon.py

```diff
@@ -24,7 +24,7 @@ def test_simple(self, safe_urlread, safe_urlopen,
             'version': {'stable': '1.0.0'},
         })
 
-        assert options.set('system.admin-email', 'foo@example.com')
+        assert options.set('system.admin-email', 'foo@example.com') is not False
```

Comment:
Oops, I can revert this and bring it back in the next pull request that will remove the default. (This was left over before putting the default cache back in.)